### PR TITLE
11 add consent form

### PR DESCRIPTION
**Pull request by Era-Dorta-TU-Delft**
This PR adds a first task that is the consent form. The user must fill in their name and then the submit button enables them to actually go on to the annotation task.

A few things to notice
1. Note that the user can still skip the consent form because the `, prerequisite=True, required=True` in the task don't do anything.
2. Also filling in the name and then deleting in also enables skipping the consent.
3. I couldn't find the original consent forms from conflab, so I created one with the help of chatgpt

I'll squash merge to keep the history clean and then I'll add the issue number.

### CODE REVIEW

**Comment by Kenneth-Funes-TU-Delft** [comment]
@Era-Dorta-TU-Delft you said "Note that the user can still skip the consent form because the , prerequisite=True, required=True in the task don't do anything." . Does that mean that the blocking still doesn't work ? or this is something you fixed ?

File: covfee/client/input/form.tsx [hunk 1]
```diff
@@ -224,7 +224,10 @@ export const Form: React.FC<React.PropsWithChildren<Props>> = (props) => {
 
       {args.renderSubmitButton && (
         <AntdForm.Item>
-          {args.renderSubmitButton({ disabled: args.disabled })}
+          {args.renderSubmitButton({
+            disabled:
+              args.disabled || (args.fields != null && args.values === null), // Don't allow submitting empty forms
```
Comment (Kenneth-Funes-TU-Delft) [inline covfee/client/input/form.tsx:229]: I'm not 100% but I think you want `args.fields !== null`

File: samples/continuous_annotation/www/consent.md
Comment (Kenneth-Funes-TU-Delft) [inline samples/continuous_annotation/www/consent.md:1]: Minor: technically, this should be more of an "End User License Agreement" (EULA) than a consent, as the priority is to grant rights to the annotator to access the data, not so much as to collect annotator data for which a consent would be appropriate. Let me check the GDrive folders for the old one

**Review by Kenneth-Funes-TU-Delft — CHANGES_REQUESTED** [review]
Looks good, but please change the consent to the proper EULA

**Comment by Era-Dorta-TU-Delft** [comment]
I've address all the points

**Review by Kenneth-Funes-TU-Delft — APPROVED** [review]
Thanks!

**Pull request merged**